beta-src/src/state/game/game-api-slice.ts

```diff
@@ -14,7 +14,7 @@ export const fetchGameOverview = createAsyncThunk(
       ApiRoute.GAME_OVERVIEW,
       queryParams,
     );
-    return data as GameOverviewResponse;
+    return data.data as GameOverviewResponse;
```

Comment:
booooooooooo. do this ugliness in the deconstruct so that this line remains as return data as ...

Reply:
Where would I deconstruct it?

Reply:
line 13 above this line. i see why you used data it makes sense axios does that too...this was a really nit picky comment...but i do think it would look cleaner this way.